backend/src/graphql/resolver/ContributionResolver.test.ts

```diff
@@ -706,627 +591,1031 @@ describe('ContributionResolver', () => {
-                    state: 'CONFIRMED',
-                    memo: 'Herzlich Willkommen bei Gradido!',
-                    amount: '1000',
-                  }),
-                  expect.objectContaining({
-                    id: expect.any(Number),
-                    state: 'PENDING',
-                    memo: 'Test env contribution',
-                    amount: '100',
-                  }),
-                ]),
-              },
+          })
+
+          await mutate({
+            mutation: login,
+            variables: { email: 'peter@lustig.de', password: 'Aa12345_' },
+          })
+
+          await mutate({
+            mutation: denyContribution,
+            variables: {
+              id: contribution.data.createContribution.id,
             },
-          }),
-        )
+          })
+
+          const { errors: errorObjects }: { errors: GraphQLError[] } = await mutate({
+            mutation: denyContribution,
+            variables: {
+              id: contribution.data.createContribution.id,
+            },
+          })
+          expect(errorObjects).toMatchObject([new GraphQLError('Contribution not found')])
+        })
+
+        it('logs the error found', () => {
+          expect(logger.error).toBeCalledWith(`Contribution not found`, expect.any(Number))
+        })
       })
 
-      it('returns all IN_PROGRESS Creation', async () => {
-        await expect(
-          query({
-            query: listAllContributions,
+      describe('valid input', () => {
+        it('deny contribution', async () => {
+          await mutate({
+            mutation: login,
+            variables: { email: 'peter@lustig.de', password: 'Aa12345_' },
+          })
+          const {
+            data: { denyContribution: isDenied },
+          }: { data: { denyContribution: boolean } } = await mutate({
+            mutation: denyContribution,
             variables: {
-              currentPage: 1,
-              pageSize: 25,
-              order: 'DESC',
-              statusFilter: ['IN_PROGRESS'],
+              id: contributionToDeny.data.createContribution.id,
             },
-          }),
-        ).resolves.toEqual(
-          expect.objectContaining({
-            data: {
-              listAllContributions: {
-                contributionCount: 0,
-                contributionList: expect.not.arrayContaining([
-                  expect.objectContaining({
-                    id: expect.any(Number),
-                    state: 'CONFIRMED',
-                    memo: 'Herzlich Willkommen bei Gradido!',
-                    amount: '1000',
-                  }),
-                  expect.objectContaining({
-                    id: expect.any(Number),
-                    state: 'PENDING',
-                    memo: 'Test env contribution',
-                    amount: '100',
-                  }),
-                ]),
-              },
+          })
+          expect(isDenied).toBeTruthy()
```

Comment:
In general, here we want 
```suggestion
          expect(isDenied).toBe(true)
```
As we expect `true`. Else results like `'string'`, `{}`, `[]` are to be truthy as well. `toBeTruthy()` means, that the value evaluates to true

Reply:
OK thanks for explication!